src/installation/guides/fullusb.md

```diff
@@ -0,0 +1,45 @@
+# Live USB To Full Install
+
+This installation guide shows how to convert a Live USB of Void Linux into a
+Full Pervasive Install. It's recommended to use a machine with at least 4GB of
+RAM, and a USB with at least 4GB of storage to perform the installation.
+
+## Setup
+
+To begin [download](./../downloading.md) a Void Live ISO and [write it to a
+USB](./../live-images/prep.md).
+
+To modify the same device that the Operating System was booted from the root
+filesystem must first be placed elsewhere, in this case the RAM. This can be
+done from the Live USB by choosing the RAM option when prompted at boot:
```

Comment:
"In order to modify the device that the operating system was booted from, the root filesystem must first be placed elsewhere. In our case, it will be placed in RAM. This can be done ...."